### PR TITLE
Benefit eligibility questionnaire results page

### PR DESCRIPTION
**Pull request by vetskrieg**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

If the folder you changed contains a `manifest.json`, search for its `entryName` in the content-build [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) (the `entryName` there will match).

If an entry for this folder exists in content-build and you are:
1. **Deleting a folder**: Delete the application entry in [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) and merge that PR **before** this one
- _Add the link to your merged content-build PR here_

2. **Renaming or moving a folder**: Update the entry in the [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json), but do not merge it until your vets-website changes here are merged. The content-build PR must be merged immediately after your vets-website change is merged in to avoid CI errors with content-build (and Tugboat).

### :warning: TeamSites :warning:
Examples of a TeamSite: https://va.gov/health and https://benefits.va.gov/benefits/. This scenario is also referred to as the "injected" header and footer. You can reach out in the `#sitewide-public-websites` Slack channel for questions.

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

Adding a results page for benefit eligibility questionnaire.

## Related issue(s)

[PTEMSVT-86](https://jira.devops.va.gov/browse/PTEMSVT-86)

## Testing done

Local testing.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  ||image|
| Desktop ||image|

## What areas of the site does it impact?

The benefit eligibility questionnaire which is disabled.

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/applications/benefit-eligibility-questionnaire/components/BenefitCard.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <p className="vads-u-margin-y--0">
+      GI Bill benefits help you pay for college, graduate school, and training
+      programs. Learn more about GI Bill benefits and how to apply for them.
+    </p>
+    <div>
+      <div className="vads-u-display--inline-block vads-u-margin-right--2">
+        <va-link-action
+          href="#"
+          message-aria-describedby="Learn more"
+          text="Learn more"
```
Comment (jamigibbs) [inline src/applications/benefit-eligibility-questionnaire/components/BenefitCard.jsx:18]: You won't need the extra `message-aria-describedby` props set if it's not adding any additional text that the `text` prop already has. In this case the screen reader will hear "learn more" twice in a row.

File: src/applications/benefit-eligibility-questionnaire/components/BenefitCard.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          type="secondary"
+        />
+      </div>
+      <div className="vads-u-display--inline-block">
+        <va-link-action
+          href="#"
```
Comment (jamigibbs) [inline src/applications/benefit-eligibility-questionnaire/components/BenefitCard.jsx:24]: Did you intend to handle the click event somehow with this action link and the one before too?

**Review by jamigibbs — DISMISSED** [review]

File: src/applications/benefit-eligibility-questionnaire/components/BenefitCard.jsx (hunk 2 above)
Comment (vetskrieg) [reply src/applications/benefit-eligibility-questionnaire/components/BenefitCard.jsx]: Yes, in a future iteration.

**Review by ghost — APPROVED** [review]
Nice work!

**Review by jamigibbs — APPROVED** [review]

**Pull request merged**